### PR TITLE
Rst msw M2

### PR DESCRIPTION
**Pull request by joakim-hove**
Changes required to restart M2 without historical schedule section - work in progress.

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Comments to this? The "real stuff" has been merged already.

File: opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp [hunk 1]
```diff
@@ -22,6 +22,7 @@
 #define WELL2_HPP
 
 #include <string>
+#include <iostream>
```
[the remark this answers is not in the document]
Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp]: Changed to `include <ostream>`

[the remark this answers is not in the document]
Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp]: Changed

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (atgeirr) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp]: What about `<iosfwd>` in the header (and `<ostream>` in the cpp file)?

[the remark this answers is not in the document]
Comment (joakim-hove) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp]: OK - I have done as @atgeirr suggested

**Comment by joakim-hove** [comment]
jenkins build this please

File: opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp (hunk 1 above)
[the remark this answers is not in the document]
Comment (bska) [reply opm/parser/eclipse/EclipseState/Schedule/Well/Well.hpp]: > OK - I have done as @atgeirr suggested

Very good.  `<iosfwd>` in the header and `<ostream>` in the implementation file is the best option here.  Thanks!

**Comment by bska** [comment]
> I'll merge once the build check comes back green.

Build is green.  I'm merging this now.  Thanks a lot.

**Pull request merged**